src/segment-loader.js

```diff
@@ -303,6 +303,11 @@ export default class SegmentLoader extends videojs.EventTarget {
     if (this.state === 'READY') {
       this.fillBuffer_();
     }
+
+    if (this.checkBufferTimeout_) {
+      window.clearTimeout(this.checkBufferTimeout_);
+    }
```

Comment:
This is a separate fix. I noticed that after seeking a lot we ended up with `monitorBuffer_ ` being called ever few milliseconds instead of twice a second. Something was causing the timer to be set repeatedly.

There is no good reason not to unset the timer before creating a new one just to guard against some other code calling `monitorBuffer_`.